Library/ViewModels/PledgeShippingLocationViewModelTests.swift

```diff
@@ -18,7 +18,7 @@ private let shippingRules = [
 final class PledgeShippingLocationViewModelTests: TestCase {
   private let vm: PledgeShippingLocationViewModelType = PledgeShippingLocationViewModel()
 
-  private let amountAttributedText = TestObserver<NSAttributedString, Never>()
+  private let amountText = TestObserver<String, Never>()
```

Comment:
Since we're not really interested in any attributes I'm simply mapping the `string` value for more readable tests